xds/src/main/java/io/grpc/xds/internal/sds/ServerSslContextProviderFactory.java

```diff
@@ -34,8 +34,8 @@
   private final CertProviderServerSslContextProvider.Factory
       certProviderServerSslContextProviderFactory;
 
-  ServerSslContextProviderFactory() {
-    this(Bootstrapper.getInstance(), CertProviderServerSslContextProvider.Factory.getInstance());
+  ServerSslContextProviderFactory(Bootstrapper bootstrapper) {
+    this(bootstrapper, CertProviderServerSslContextProvider.Factory.getInstance());
```

Comment:
Same comment as for `ClientSslContextProviderFactory`.

Reply:
Same answer as above.